__tests__/localstorage.test.js

```diff
@@ -0,0 +1,147 @@
+describe('tests', () => {
+  test('add the task at local storage', () => {
+    // Arrange
+    const tasks = [
+      {
+        description: 'task 1',
+        complete: false,
+        index: 1,
+      },
+      {
+        description: 'task 2',
+        complete: false,
+        index: 2,
+      },
+    ];
+    // Mock function
+    const mockAddLS = jest.fn((tasks, descrp) => {
+      const tElement = {
+        description: descrp,
+        complete: false,
+        index: tasks.length + 1,
+      };
+      tasks.push(tElement);
+      return tasks.length;
+    });
+    // Act
+    expect(mockAddLS(tasks, 'New descrp')).toBe(3);
+  });
+  test('delet the task at local storage', () => {
+    // Arrange
+    const tasks = [
+      {
+        description: 'task 1',
+        complete: false,
+        index: 1,
+      },
+      {
+        description: 'task 2',
+        complete: false,
+        index: 2,
+      },
+    ];
+    // Mock function
+    const mockDeletTask = jest.fn((tasks, index) => {
+      tasks.splice(index, 1);
+      return tasks.length;
+    });
+    // Act
+    expect(mockDeletTask(tasks, 0)).toBe(1);
+  });
```

Comment:
- [ ] Good job on wrapping the tests inside the describe 👍 however you are not actually testing the code you've written, but just the mock function you've written inside of your tests.